modules/nf-scil/registration/tractogram/main.nf

```diff
@@ -0,0 +1,83 @@
+process REGISTRATION_TRACTOGRAM {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'scil.usherbrooke.ca/containers/scilus_1.6.0.sif':
+        'scilus/scilus:1.6.0' }"
+
+    input:
+    tuple val(meta), path(anat), path(transfo), path(centroids_dir), path(ref) /* optional, value = [] */
+
+    output:
+    tuple val(meta), path("*__*.trk"), emit: warped_bundle
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def reference = "$ref" ? "--reference $ref" : ""
+
+    def inverse = task.ext.inverse ? "--inverse " + task.ext.inverse : ""
```

Comment:
Be careful here these are store_true options:
1. you probably don't need to add this to the line: "--inverse " + task.ext.inverse
     def inverse = task.ext.inverse ? "--reverse " : ""
2. I think you need to put ifs for these defs (to be confirmed with Alex)

see: https://github.com/scilus/nf-scil/blob/main/modules/nf-scil/reconst/fodf/main.nf

Reply:
It is not a requirement to use `if/else` statements, the ternary operator does the job (we'll uniformize the modules in the Code freeze period that's coming). As for the boolean flags (what you called store_true), you are correct @Manonedde.

On that matter @anroy1, I want you to be extra careful with those. It's a mistake I've been seeing on a lot of your submitted PRs.